playbook.yml

```diff
@@ -61,9 +66,15 @@
       src: files/config.json
       dest: /root/web-platform-tests/
 
+  - name: Collect list of host aliases
+    shell: /root/web-platform-tests/wpt make-hosts-file
+    register: hosts_file_contents
+
   - name: Update the hosts file
-    shell: /root/web-platform-tests/wpt make-hosts-file | sudo tee -a /etc/hosts
-    become: true
+    lineinfile:
+      line: '{{item}}'
+      dest: /etc/hosts
+    with_items: '{{hosts_file_contents.stdout_lines}}'
```

Comment:
I like this change. When I tested locally it printed the output so I could verify the correct hosts files were written.